hieradata/dev/wso2/wso2esb/4.9.0/mesos/manager.yaml

```diff
@@ -29,19 +23,24 @@ wso2::clustering:
     marathon_endpoint: http://marathon.mesos:8080/v2/
     mesos_applications: wso2esb-manager,wso2esb-worker
```

Comment:
Thanks Imesh. What I actually wanted to check was, the hostname and mgt_hostname will resolve. The hostname will be used for internal communication. We might need to check and fix this.